timer-selector.js

```diff
@@ -479,74 +527,128 @@ function buildTimer(language) {
     data.styling,
   );
 
-  outputElement.value = buildWikitext(data.template, url, data.height, data.width);
+  outputElement.value = buildWikitext(language, data.template, url, score, data.height, data.width);
+
+  if (copyToClipboard) {
+    navigator.clipboard.writeText(outputElement.value);
+  }
+}
+
+function setMessage(language, id, messageKey = null) {
+  document.getElementById(id).innerText = getMessage(language, messageKey || id);
+}
+
+function initializeSummaryDeletionMessages(language) {
+  // Summary deletion reasons vary by site
+  var summaryDeletionBox = document.getElementById('summary-deletion-reason');
+  for (var i = 1; i < 25; ++i) {
```

Comment:
Why is this set to 25 rather than having each language have its own set of reasons?

Reply:
I set it to a silly high number because the test language would continue adding options indefinitely otherwise.